assets/js/modules/analytics-4/components/setup/SetupEnhancedMeasurementSwitch.test.js

```diff
@@ -176,46 +169,6 @@ describe( 'SetupEnhancedMeasurementSwitch', () => {
 		).not.toBeInTheDocument();
 	} );
 
-	it.each( [
-		[ 'propertyID', PROPERTY_CREATE ],
-		[ 'webDataStreamID', WEBDATASTREAM_CREATE ],
-	] )(
```

Comment:
Not sure why we had these two to begin with, as based on original [AC](https://github.com/google/site-kit-wp/issues/7459) and discussion with @techanvil we want to be pretty pushy on setup component page, and switch should always get turned on by default, on load and when property/web data stream is switched, regardless is it creating a new one or the one where previously user had EM disabled. They are not fitting the designed behaviour, so I removed them

Reply:
Hey @zutigrm, I've been trying to figure this out - I think this test might be there to cover the scenario where the user has just returned from OAuth having chosen to create a new property/data stream with the toggle in the off position, in which case we do want to honour the state of `isEnhancedMeasurementEnabled`.

I should have provided a bit more context in the test description but this seems quite likely to be the reason for it.

So, please do be careful that scenario is still working as expected, and reconsider keeping this test.

Reply:
Thanks @techanvil . Yes, this issue is about ensuring that behaviour works as intended, as the issue was that it was always overriding it. The switch to reset the EM value from the drop downs itself is correcting this behaviour as user's choice will be respected unless he changes account/property/web data stream dropdown, only in that case value will be reset

Reply:
Ahh, right. I had got a bit mixed up and thought that was happening in the Settings flow but I see it's happening for the Setup flow. Thanks for clarifying!